package.json

```diff
@@ -33,5 +20,27 @@
     "Fabien O'Carroll <fabien@allou.is>",
     "Kirill Fomichev <fanatid@ya.ru>"
   ],
-  "license": "MIT"
+  "files": [
+    "index.js"
```

Comment:
Agreed, re-added here 58256cf0debfcfcb95c7427fc480847213440ce1 along with `scripts` and `benchmarks`. My thinking here is that I should be able to `npm install` and run whatever script is listed in `package.json` without having to checkout the repo.